PLUGINS.md

```diff
@@ -225,7 +225,7 @@ to retrieve your plugin's settings in a platform-independent way.
 **Be sure to use a unique prefix for any settings you save so as not to clash
 with core EDMC or other plugins.**
 
-Use `numberFromString()` from EDMC's `l10n.Locale` object to parse input
+Use `number_from_string()` from EDMC's `l10n.Locale` object to parse input
```

Comment:
Urgh, I know it's the standard, but this counts as changing a published API.  That means bumping major version on the first release with this in.

On those grounds it would be better to keep the old name for the time being.

Reply:
Ah, you attempted to just make the old names deprecated, but see comments on those bits, it looks like you brainfarted and have introduced infinite recursion if the old names are used.

Reply:
Yep! fixed now

Reply:
Maybe explicitly document here that using the old name will cause deprecation logging *but continue to work for the time being*.  Just so no-one screams about the change, despite it only causing log noise at this point.